app/views/static/site_map.html.erb

```diff
@@ -116,11 +116,11 @@
         <% if CurrentUser.is_admin? %>
           <li><%= link_to("Admin Dashboard", admin_dashboard_path) %></li>
           <li><%= link_to("Email Blacklist", email_blacklists_path) %></li>
-          <li><%= link_to("Upload Whitelist", upload_whitelists_path) %></li>
           <li><%= link_to("Post Report Reasons", post_report_reasons_path) %></li>
           <li><%= link_to("Alias & Implication Import", new_admin_alias_and_implication_import_path) %></li>
           <li><%= link_to("Danger Zone", admin_danger_zone_index_path) %></li>
         <% end %>
+          <li><%= link_to("Upload Whitelist", upload_whitelists_path) %></li>
```

Comment:
Not properly indented. Also, please move it down to the other member visible links, just so they are more grouped together in the code.